lib/generateMarkdown.js

```diff
@@ -74,20 +72,31 @@ function generateMarkdown(options) {
 
         // Render section for each property
         md += createPropertiesDetails(schema, title, options.headerLevel + 1, options.suppressWarnings);
+        md += createPropertiesDetails(schema, title, options.headerLevel + 1, options.suppressWarnings, 'definitions');
     }
 
     return md;
 }
 
 ////////////////////////////////////////////////////////////////////////////////
 
-function createPropertiesSummary(schema, suppressWarnings) {
+function createPropertiesSummary(schema, suppressWarnings, ptype) {
```

Comment:
If you're updating the function definition, this is a good time to add a function header that explains what each parameter is, especially with an ambiguous name like `ptype`.